operations/mimir/autoscaling.libsonnet

```diff
@@ -122,11 +122,6 @@
             serverAddress: $._config.autoscaling_prometheus_url,
             query: trigger.query,
 
-            // The metric name uniquely identifies a metric in the KEDA metrics server.
-            // This is deprecatd in KEDA 2.10, and is instead set to a default based on the trigger type (e.g. `s0-prometheus`).
-            // Instead we use the metric_name as the name for the trigger (above). This appears as the `scaler` label on `keda_scaler_metrics_value`.
-            metricName: trigger.metric_name,
```

Comment:
should we switch to `name` instead of `metricName` now? https://keda.sh/docs/2.16/reference/scaledobject-spec/#triggers